include/small/rlist.h

```diff
@@ -31,6 +31,7 @@
  * SUCH DAMAGE.
  */
 #include <stddef.h>
+#include <stdbool.h>
```

Comment:
We need it for expressions like this: `(((tmp) = rlist_prev_entry_safe((item), (head), member)) || true);`. We need `|| true` part here because `rlist_prev_entry_safe` would return `NULL` in the end so `((tmp) = rlist_prev_entry_safe((item), (head), member))` is false for `item` being the last element (`tmp` looks into the head of the list).